packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/modules/Module.kt

```diff
@@ -12,11 +15,22 @@ abstract class Module {
   val appContext: AppContext
     get() = requireNotNull(_appContext) { "The module wasn't created! You can't access the app context." }
 
+  @Suppress("PropertyName")
+  @PublishedApi
+  internal lateinit var _coroutineScopeDelegate: Lazy<CoroutineScope>
```

Comment:
Why did you prefix it with `_`? As far as I remember, we don't do this in Kotlin yet 😅